cityofaustin/atd-moped
# Implement eCapris funding UI enhancements

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/10025

## Testing
**URL to test:** 
https://deploy-preview-766--atd-moped-main.netlify.app/moped/projects

**Steps to test:**

Add a real ecapris subproject ID to a project (ex: 7333.001, 11895.001, 13270.018)
See that the Add funding button turns into a dropdown
Test adding a generic entry
clicking "from ecapris" will open a modal with possible fdus
the add button should be disabled until an fdu is selected
Once an fdu has been added to the table, it does not appear on the modal anymore.

---
#### Ship list
